Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:23.04
```

Comment:
Can we move Dockerfile, action.yml, entrypont.sh to the tools directory?  Or to  tools/docker  ?

Reply:
I was in dilemma about it, I made them all gather in a subdirectory in tools/custom-action! Thank you very much!